buildbot.mariadb.org/scripts/docker_cleanup.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -xeuvo pipefail
+
+mariadb_version=${1}
+mariadb_version=${mariadb_version#*-}
+master_branch=${mariadb_version%\.*}
+
+commit=${2:-}
+branch=${3:-${master_branch}}
+
+if [[ $branch =~ ^preview ]]; then
+  container_tag=${branch#preview-[0-9]*\.[0-9]*-}
+else
+  container_tag=$master_branch
+fi
+# Container tags must be lower case.
+container_tag=${container_tag,,*}
+
+t=$(mktemp)
+for m in mariadb-devel-${container_tag}-$commit mariadb-debug-${container_tag}-$commit
+do
+  echo cleanup $m
+  buildah manifest inspect "$m" | jq '.manifests[]?.digest?' | tee -a "$t"
+done
+
+podman images --filter dangling=true --format '{{.ID}} {{.Digest}}' |
```

Comment:
Woudln't `podman image prune -f` be able to replace this whole while loop?

Reply:
no, we're removing only those image id that where in the digest.
During the build there are untagged images that are part of the manifest.

Reply:
Right, I didn't notice the "grep... $t"